app/components/new-directory-user.hbs

```diff
@@ -1,292 +1,320 @@
 {{! template-lint-disable no-attrs-in-components }}
```

Comment:
Are these still needed? Can the need for them be removed in this refactor?

Reply:
:heavy_check_mark: